charts/velero/templates/upgrade-crds.yaml

```diff
@@ -52,6 +52,8 @@ spec:
         - name: kubectl
           {{- if .Values.kubectl.image.digest }}
           image: "{{ .Values.kubectl.image.repository }}@{{ .Values.kubectl.image.digest }}"
+          {{- else if .Capabilities.KubeVersion.Version }}
```

Comment:
Do we need to document this logic in `values.yaml`?

Seems I have no way to use the  customized image specified by `.Values.kubectl.image.tag`

How about processing the image in order `digest`>`tag`>`Capabilities.KubeVersion.Version` and make the `.Values.kubectl.image.tag` commented by default?

Reply:
> How about processing the image in order `digest`>`tag`>`Capabilities.KubeVersion.Version` and make the `.Values.kubectl.image.tag` commented by default?

Good suggestion.

> Do we need to document this logic in `values.yaml`?

Sure.

> 
> Seems I have no way to use the customized image specified by `.Values.kubectl.image.tag`
> 
> How about processing the image in order `digest`>`tag`>`Capabilities.KubeVersion.Version` and make the `.Values.kubectl.image.tag` commented by default?

Good suggestion.